library/swift/src/EngineImpl.swift

```diff
@@ -9,8 +9,8 @@ final class EngineImpl: NSObject {
   private let streamClientImpl: StreamClientImpl
 
   private enum ConfigurationType {
-    case yaml(String)
-    case typed(EnvoyConfiguration)
+    case custom(String, EnvoyConfiguration)
+    case standard(EnvoyConfiguration)
```

Comment:
```suggestion
    case custom(yaml: String, config: EnvoyConfiguration)
    case standard(config: EnvoyConfiguration)
```

It might be nice to name these now since `String` is ambiguous, but not a big deal since this is private